app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventFragment.java

```diff
@@ -118,6 +108,7 @@ public void onStart() {
         getPresenter().attach(this);
         binding.setEvent(getPresenter().getEvent());
         getPresenter().start();
+        ((AppCompatActivity)getActivity()).getSupportActionBar().hide();
```

Comment:
Why this?

Reply:
Actually I wanted the action bar to be hidden so that there could be no navigation possible directly from this CreateEventFragment , hence I hid it.

Reply:
But it should be possible and there should be a back button as well